src/originalBudgetTrackingApp/commonTypes.ts

```diff
@@ -1,7 +1,31 @@
 import { Transaction } from '@/originalBudgetTrackingApp/bankScraper';
+import { CompanyTypes } from '@brafdlog/israeli-bank-scrapers-core';
+
+export interface ProgressEmitter {
+  onProgress(func: (...args: any[]) => void): void
+}
```

Comment:
To expect [this method](https://github.com/eshaham/israeli-bank-scrapers/blob/83c4b664137e9bfdd76d8ef71cdce47a55733f58/src/scrapers/base-scraper.ts#L192-L194) in the `israeli-bank-scrapers`